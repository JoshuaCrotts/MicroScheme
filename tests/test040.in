(define x (+ 2 3))
(true? (< x 5))
(false? (= x 5))
(true? (> x 5))
(not (true? (= x 5)))
(false? (not (true? (= x 5))))
