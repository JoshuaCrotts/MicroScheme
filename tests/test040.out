#f
#f
#f
#f
#t
